packages/cli/assets/pluginPackage/src/modelDefine.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  ModelDefineType,
+  UniDataset,
+  ModelDefineArgsType,
+  UniModel,
+  Sample
+} from '@pipcook/pipcook-core';
+
+const templateModelDefine: ModelDefineType = async (data: UniDataset, args: ModelDefineArgsType): Promise <UniModel> => {
+
+  let model = {}; // Some model defination
+
+  const retModelDefine: UniModel = {
+    model,
+    predict: async function (input: Sample) {
```

Comment:
```suggestion
    predict: async (input: Sample) => {
```

Reply:
Should bind this.

Reply:
It's not necessary to use an arrow function here, the `this` is a reference to `retModelDefine`, which is correct.

Reply:
+1; I think the two forms here are identical.

Reply:
Yeah, it' correct.